Doc/library/imaplib.rst

```diff
@@ -353,7 +360,7 @@ An :class:`IMAP4` instance has the following methods:
    Send ``NOOP`` to server.
 
 
-.. method:: IMAP4.open(host, port)
+.. method:: IMAP4.open(host, port, timeout)
```

Comment:
`timeout` needs to be optional. Again, using `timeout=None`. It also needs a description of `timeout`.